docs/permissions/getting-started.md

```diff
@@ -44,118 +30,103 @@ Asset requests initiated by the browser will not include a token in the `Authori
-
-   ```typescript title="packages/backend/src/plugins/permission.ts"
-   import { createRouter } from '@backstage/plugin-permission-backend';
-   import {
-     AuthorizeResult,
-     PolicyDecision,
-   } from '@backstage/plugin-permission-common';
-   import { PermissionPolicy } from '@backstage/plugin-permission-node';
-   import { Router } from 'express';
-   import { PluginEnvironment } from '../types';
-
-   class TestPermissionPolicy implements PermissionPolicy {
-     async handle(): Promise<PolicyDecision> {
-       return { result: AuthorizeResult.ALLOW };
-     }
-   }
-
-   export default async function createPlugin(
-     env: PluginEnvironment,
-   ): Promise<Router> {
-     return await createRouter({
-       config: env.config,
-       logger: env.logger,
-       discovery: env.discovery,
-       policy: new TestPermissionPolicy(),
-       identity: env.identity,
-     });
-   }
-   ```
-
-3. Wire up the permission policy in `packages/backend/src/index.ts`. [The index in the example backend](https://github.com/backstage/backstage/blob/master/packages/backend/src/index.ts) shows how to do this. You’ll need to import the module from the previous step, create a plugin environment, and add the router to the express app:
-
-   ```ts title="packages/backend/src/index.ts"
-   import proxy from './plugins/proxy';
-   import techdocs from './plugins/techdocs';
-   import search from './plugins/search';
-   /* highlight-add-next-line */
-   import permission from './plugins/permission';
-
-   async function main() {
-     const techdocsEnv = useHotMemoize(module, () => createEnv('techdocs'));
-     const searchEnv = useHotMemoize(module, () => createEnv('search'));
-     const appEnv = useHotMemoize(module, () => createEnv('app'));
-     /* highlight-add-next-line */
-     const permissionEnv = useHotMemoize(module, () => createEnv('permission'));
-     // ..
-
-     apiRouter.use('/techdocs', await techdocs(techdocsEnv));
-     apiRouter.use('/proxy', await proxy(proxyEnv));
-     apiRouter.use('/search', await search(searchEnv));
-     /* highlight-add-next-line */
-     apiRouter.use('/permission', await permission(permissionEnv));
-     // ..
-   }
-   ```
+1. Add `@backstage/plugin-permission-backend` and `@backstage/plugin-permission-backend-module-allow-all-policy` to your backend dependencies, this will add the permission backend and a policy that allows all permissions:
+
+```bash
+# From your Backstage root directory
+yarn --cwd packages/backend add @backstage/plugin-permission-backend
+```
+
+```bash
+# From your Backstage root directory
+yarn --cwd packages/backend add @backstage/plugin-permission-backend-module-allow-all-policy
+```
+
+2. Add the following to `packages/backend/src/index.ts`. This adds the permission-backend router, and configures it with a policy which allows everything.
```

Comment:
Reading through your instructions while working on permissions migration. What are your thoughts on combining this into a single yarn command?